docker/Dockerfile

```diff
@@ -30,19 +30,15 @@ RUN \
 		imagemagick \
 		gsfonts
 
-WORKDIR /tmp/
-
-RUN \
-	echo "Downloading newer version of Cairo" \
-	&& curl https://www.cairographics.org/releases/cairo-1.14.8.tar.xz | tar xvfJ - -C .
-
-WORKDIR /tmp/cairo-1.14.8/
-
-RUN \
-  echo "Installing newer version of Cairo" \
-	&& ./configure \
-	&& make \
-	&& make install
+RUN echo "Install libcairo2 from stretch" \
+	&& echo 'APT::Default-Release "stretch";' > /etc/apt/apt.conf \
```

Comment:
https://github.com/aquavitae/docker-weasyprint/pull/5/files

by the looks of it, this can be `'APT::Default-Release "jessie";'`

then the line below goes from `stable.list` to `jessie.list` and the line below that goes from `testing.list` to `stretch.list`

😵